src/bridge/cockpitpackages.c

```diff
@@ -600,7 +620,7 @@ maybe_add_package (GHashTable *listing,
 
   path = g_build_filename (parent, name, NULL);
 
-  manifest = read_package_manifest (path, name);
+  manifest = read_package_manifest (path, name, bundle_checksum);
```

Comment:
I think this isn't enough to get the desired effect: ~25-50 lines down from here is a section of code that calculates the `own_checksum` over the files in the directory (and only those files) and uses the existing bundle checksum in case nothing in that directory changed.  I'm pretty sure this would specifically prevent the checksum being updated in the case an override was added.

Reply:
Well spotted, thanks! Changed to what you suggested.